README.md

```diff
@@ -332,7 +332,7 @@ as follows:
           {
             home-manager.useGlobalPkgs = true;
             home-manager.useUserPackages = true;
-            home-manager.users.user = import ./home.nix;
+            home-manager.users.jdoe = import ./home.nix;
```

Comment:
Why not use "john-doe" in its entirety?

Reply:
https://github.com/nix-community/home-manager/search?q=jdoe vs https://github.com/nix-community/home-manager/search?q=jon-doe